fe/fe-core/src/main/java/com/starrocks/sql/analyzer/DataCacheStmtAnalyzer.java

```diff
@@ -164,6 +165,17 @@ public Void visitDataCacheSelectStatement(DataCacheSelectStatement statement, Co
             }
             statement.setTTLSeconds(ttlSeconds);
 
+            statement.setNumReplicasDesired(Integer.parseInt(
+                    properties.getOrDefault("num_replicas", "1")));
+            if (statement.getNumReplicasDesired() < 1) {
+                throw new SemanticException("Num replicas must be positive");
+            }
+
+            String resourceIsolationGroupsString = properties.getOrDefault("resource_isolation_groups",
```

Comment:
At this layer of abstraction, I'm not trying to decide any logic for statement execution, just to parse the input properties as strings. 

As for when we do enforce the logic, we don't want to assume the default resource isolation group is the intent. In fact, there's no guarantee that the cluster has any CN in the default resource isolation group, so that would be wrong. If the resource isolation group is not supplied as part of the statement, we assume that the user sent the statement to the FE of the intended resource isolation group.